src/libraries/System.Private.CoreLib/src/System/AppContext.cs

```diff
@@ -82,6 +82,28 @@ public static void SetData(string name, object? data)
         internal static event EventHandler<FirstChanceExceptionEventArgs>? FirstChanceException;
 #pragma warning restore CS0067
 
+        internal static bool OnUnhandledExceptionWorker(object e)
```

Comment:
This `bool` value is actually useless. It is used as part of dead code. I am deleting the dead code in #114419. Could you please change it to return void in this PR?